vlog4j-parser/src/test/java/org/semanticweb/vlog4j/syntax/parser/RuleParserTest.java

```diff
@@ -412,6 +354,27 @@ public void testBlankPredicateName() throws ParsingException {
 		RuleParser.parse(input);
 	}
 
+	@Test(expected = ParsingException.class)
+	public void testInvalidDatatypeOnLiteral() throws ParsingException {
```

Comment:
this is more like an invalid Literal string, instead of an invalid datatype on literal